site/app/libraries/database/DatabaseQueries.php

```diff
@@ -256,6 +256,33 @@ public function getAllUnarchivedSemester() {
         return $return;
     }
 
+    /**
+     * @return string[]
+     */
+    public function getAllTerms() {
+        $this->submitty_db->query(
+            "SELECT term_id FROM terms ORDER BY start_date DESC"
+        );
+        $return = array();
+        foreach ($this->submitty_db->rows() as $row) {
+            $return[] = $row['term_id'];
+        }
+        return $return;
+    }
+
+    /**
+     * @param string $term_id
+     * @param string $term_name
+     * @param date $start_date
+     * @param date $end_date
+     */
+    public function createNewTerm($term_id, $term_name, $start_date, $end_date) {
+        $this->submitty_db->query(
+            "INSERT INTO terms (term_id, name, start_date, end_date) VALUES (?, ?, ?, ?)",
+            array($term_id, $term_name, $start_date, $end_date)
```

Comment:
Please switch `array()` syntax to use `[]` syntax for the instantiation of arrays

Reply:
okay, I'll. But can i understand the reason behind that?

Reply:
As of PHP 5.4 we can use the short syntax for arrays and it is preferred.

Example (taken from https://php.net/manual/en/language.types.array.php):
```
<?php
$array = array(
    "foo" => "bar",
    "bar" => "foo",
);

// as of PHP 5.4
$array = [
    "foo" => "bar",
    "bar" => "foo",
];
?>
```